nipy/algorithms/graph/graph.py

```diff
@@ -680,7 +680,7 @@ def dijkstra(self, seed=0):
         dist, active = np.inf * np.ones(self.V), np.ones(self.V)
         idx, neighb, weight = self.compact_neighb()
         dist[seed] = 0
-        dg = zip(np.zeros_like(seed), seed)
+        dg = list(zip(np.zeros_like(seed), seed))
```

Comment:
In terms of performance, listing a list is really trivial (a few microseconds).   The reason I like it is because, in the next few years where Python 3 takes over, it's clearer what is intended - more Python 3-ey and less six-ey.